build/docker/zap_common.py

```diff
@@ -374,6 +374,10 @@ def get_latest_zap_client_version():
 
 
 def check_zap_client_version():
+    if running_in_docker():
+        logging.info('No need to check ZAP Python API client while running in Docker')
```

Comment:
I'd not log at all, if you prefer log something I'd suggest debug instead of info, it does not add much for the usage of the script in Docker.

Reply:
Done